public/app/object-cache.php

```diff
@@ -1238,7 +1238,9 @@ protected function _connect_redis() {
 	 *               not with a message describing the issue.
 	 */
 	public function check_client_dependencies() {
-		if ( ! class_exists( 'Redis' ) ) {
+		// *
+		// Check for the correct Redis extension.
+		if ( ! class_exists( WP_REDIS_USE_RELAY ? 'Relay\Relay' : 'Redis' ) ) {
```

Comment:
Hey mate. Are we not decided on the use here? Or, is this a local consideration?